.azure-pipelines/pipelines.yml

```diff
@@ -337,7 +337,9 @@ stages:
               itemPattern: "docker/envoy-docker-images.tar.xz"
               downloadType: single
               targetPath: $(Build.StagingDirectory)
-          - bash: ./ci/do_ci.sh verify_examples
+          - bash: |
+              apt-get update -y -qq && apt-get install -y -qq --no-install-recommends expect
+              ./ci/do_ci.sh verify_examples
```

Comment:
@mattklein123 i removed the wrapper as it wasnt really necessary, and altho it brought `expect` using it also created its own problems

ive put the apt-get here as this is the only place i think it can be put without tying the `verify_examples.sh` script to deb-based systems

Reply:
hmm - actually scrap that - i just realised that do_ci already has apt-get - moving it...